src/android/SplashScreen.java

```diff
@@ -301,16 +305,9 @@ public void run() {
                 splashImageView.setMinimumWidth(display.getWidth());
 
                 // TODO: Use the background color of the webView's parent instead of using the preference.
-                splashImageView.setBackgroundColor(preferences.getInteger("backgroundColor", Color.BLACK));
+                splashImageView.setBackgroundColor(Color.parseColor(preferences.getString("SplashBackgroundColor", "#FFFFFF")));
 
-                if (isMaintainAspectRatio()) {
-                    // CENTER_CROP scale mode is equivalent to CSS "background-size:cover"
-                    splashImageView.setScaleType(ImageView.ScaleType.CENTER_CROP);
-                }
-                else {
-                    // FIT_XY scales image non-uniformly to fit into image view.
-                    splashImageView.setScaleType(ImageView.ScaleType.FIT_XY);
-                }
+                splashImageView.setScaleType(ImageView.ScaleType.valueOf(getSplashScaleType()));
```

Comment:
What happens if the user typed in a wrong key? Maybe catch the thrown error, display a warning and fallback to default??